client/signup/steps/theme-selection/index.jsx

```diff
@@ -124,15 +128,22 @@ class ThemeSelectionStep extends Component {
 			return null;
 		}
 
+		const storeSignup = this.isStoreSignup();
```

Comment:
I think we prefer to prefix variables/functions names with `is` if it holds a boolean value.